substrate/utils/frame/benchmarking-cli/src/machine/reference_hardware.json

```diff
@@ -1,7 +1,7 @@
 [
 	{
 		"metric": "Blake2256",
-		"minimum": 783.27
+		"minimum": 968.31
```

Comment:
Is no fluctuation expected?

Reply:
The benchmarks have 10% leeway per default but can be adjusted via CLI command.